braze/src/main/java/com/tealium/remotecommands/braze/BrazeRemoteCommand.java

```diff
@@ -215,7 +215,7 @@ protected void onInvoke(Response response) throws Exception {
 
     private String[] splitCommands(JSONObject payload) {
         String commandString = payload.optString(Commands.COMMAND_KEY, "");
-        return commandString.split(BrazeConstants.SEPARATOR);
+        return commandString.trim().split(BrazeConstants.SEPARATOR);
```

Comment:
is this `trim()` strictly necessary? it does get trimmed in the `parseCommands` method too

On a related note - although I've not tested it, most of the other remote commands have similar splitting and trimming already available, but does the change in the `tealium-kotlin` PR have any backwards compatibility repercussions to switching to joining using both a comma and a space? (do we need the extra space?) 
and would this also require the `SEPARATOR` to be updated across all the other commands?